docs/roles/secretary.rst

```diff
@@ -9,22 +9,33 @@ This page explains more about the responsibilities and role of the secretary.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
-
+The secretary is responsible for scheduling, advertising, and recording RITlug general and eboard meetings.
 
 Eligibility requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-< basic eligiblity requirements – what if someone is on co-op? away from campus? working remote? is under academic suspension (i.e. low GPA)? >
+* Must be a full time student on main campus for at least one semester
+* Cannot be on academic probation
+* Must be present for **75% of weekly club meetings and eboard meetings**
 
 
 Responsibilities
 ----------------
 
-< responsibilities and routine work that someone in this position is directly accountable for – what do they do and what should other eboard members expect from this person? >
+* *Weekly Presentations*
+   * Review of presentations
+   * Addition of presentations to RITlug website
+* Record meeting minutes during both general and eboard meetings
+* Setup live stream or video conference for meetings and presentations that require remote participants or viewers
```

Comment:
This is something we might have additional resources available from MAGIC to do this. I think it's a good addition and hopefully we can act on it.

Reply:
Super, *super* small thing, but "setup" should be "set up."

Reply:
I recall in the past, and in other clubs, that live streaming is something occasionally done for remote viewers. I'm not sure how frequent/if at all it's been done in the years past at RITlug, but it would be an option I could pursue.

Reply:
We have definitely done this in the past. I believe it simply hasn't been done recently because there hasn't been anyone who has requested live streaming / recording.